tests/unit/ReferenceListTest.php

```diff
@@ -52,7 +52,7 @@ public function getConstructorArg() {
 	public function testCanConstructWithReferenceListObject() {
 		$original = new ReferenceList( array( new Reference( array( new PropertyNoValueSnak( 1 ) ) ) ) );
 		$copy = new ReferenceList( $original );
-		$this->assertSame( 1, $copy->count() );
+		$this->assertEquals( $original, $copy );
```

Comment:
This uses PHP's dark magic of comparing objects. I wanted to avoid this when I wrote the test.